CHANGELOG.md

```diff
@@ -60,6 +60,8 @@ Main (unreleased)
 
 - `prometheus.exporter.snowflake` dependency has been updated to 20251016132346-6d442402afb2, which updates data ownership queries to use `last_over_time` for a 24 hour period. (@dasomeone)
 
+- Schedule new path targets faster in `loki.source.file`. (@kalleep)
```

Comment:
I'd say that it would be more of a bugfix where we fix the issue where there can be a big lag when scheduling / rescheduling a large number of files.